easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-2013.5.1-ictce-5.4.0.eb

```diff
@@ -0,0 +1,22 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+##
+name = 'GAMESS-US'
+version = '2013.5.1'
+
+homepage = 'http://www.msg.ameslab.gov/gamess/'
+description = """The General Atomic and Molecular Electronic Structure System (GAMESS)
+is a general ab initio quantum chemistry package."""
+
+toolchain = {'name': 'ictce', 'version': '5.4.0'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = [
+    'http://www.msg.chem.iastate.edu/GAMESS/download/dist.source.shtml',  # GAMESS-US sources
+]
+sources = [SOURCE_TAR_GZ]
+patches = ['compddi.patch', 'rungms.patch']
```

Comment:
these patches need to be renamed to something like `GAMESS-US-2013.5.1_compddi-max.patch` and `GAMESS-US-2013.5.1_rungms-slurm.patch`